Utilities/Pipelines/Tasks/android-build.yml

```diff
@@ -0,0 +1,25 @@
+parameters:
+  - name: platform
+    type: string
+  - name: configuration
+    type: string
+
+steps:
+  - template: checkout.yml
+
+  - bash: ${ANDROID_HOME}/tools/bin/sdkmanager --install "ndk;22.0.7026061"
```

Comment:
If we switch the Android build to a `windows-2019` agent, this will have to change as well to something like the below, which is what we have for XAL:

```yml
- task: CmdLine@2
    displayName: Install the required Android NDK
    inputs:
      targetType: inline
      script: '%ANDROID_HOME%\tools\bin\sdkmanager --install "ndk;22.0.7026061"'
```

Reply:
Ya, have fun doing this if you do this post this PR